source/wp-content/themes/wporg-developer-2023/src/version-select/style.scss

```diff
@@ -16,24 +54,29 @@
 		white-space: nowrap;
 		width: 100%;
 		text-overflow: ellipsis;
-		padding: 0.8em 1.25em 0.8em 1em;
-		border-radius: 2px;
+		padding-right: calc($arrow-icon-width + $padding-horizontal);
+
+		&:focus-visible {
+			outline-color: var(--wp--preset--color--blueberry-1);
+		}
 	}
 
-	&::after {
-		content: "";
-		display: block;
-		position: absolute;
-		right: 1em;
-		top: 0;
-		height: 100%;
-		width: 0.8em;
-		mask-image: url(../../images/chevron.svg);
-		mask-repeat: no-repeat;
-		mask-position: center;
-		mask-size: contain;
-		transform: rotate(180deg);
-		background-color: var(--wp--preset--color--charcoal-1);
-		pointer-events: none; // Allows clicking on the select element.
+	button {
```

Comment:
Could you set up some helper classes in the parent theme for button styles? Those classes would be able to use the `button` mixins, and then just use that in these custom blocks. That could cut down on the CSS needed in the child themes.

Reply:
I've made a start on this and updated the testing notes above.

Reply:
Parent PR is now ready for review.